app/api/v01/entities/vehicle.rb

```diff
@@ -50,6 +50,8 @@ def self.entity_name
   expose(:speed_multiplicator, documentation: { type: Float, desc: 'Deprecated, use speed_multiplier instead.' }) { |m| m.speed_multiplier }
   expose(:speed_multiplier, documentation: { type: Float })
   expose(:max_distance, documentation: { type: Integer, desc: 'Maximum achievable distance in meters' })
+  expose(:max_ride_distance, documentation: { type: Integer, desc: 'Maximum riding distance between two stops within a route in meters' })
```

Comment:
```suggestion
  expose(:max_ride_distance, documentation: { type: Integer, desc: 'Maximum riding distance between two stops within a route in meters.' })
```

Reply:
Til' now the rule was :
- single phrase no dot
- Multiple phrase dots